src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -557,13 +557,41 @@ private function addCacheSection(ArrayNodeDefinition $rootNode)
                     ->info('Cache configuration')
                     ->addDefaultsIfNotSet()
                     ->fixXmlConfig('pool')
+                    ->fixXmlConfig('adapter')
                     ->children()
+                        ->arrayNode('adapters')
+                            ->useAttributeAsKey('name')
+                            ->prototype('array')
+                                ->children()
+                                    ->scalarNode('parent')
+                                        ->info('The parent cache adapter service.')
+                                    ->end()
+                                    ->integerNode('default_lifetime')->end()
+                                    ->scalarNode('provider')
+                                        ->info('The service name to use as provider when the specified parent adapter needs one.')
+                                    ->end()
+                                    ->scalarNode('clearer')->defaultValue('cache.default_pools_clearer')->end()
+                                ->end()
+                            ->end()
+                            ->validate()
+                                ->ifTrue(function ($v) {
+                                    foreach ($v as $key => $config) {
+                                        if (empty($config['parent']) && !in_array($key, array('apcu', 'doctrine', 'filesystem', 'psr6', 'redis'))) {
+                                            return true;
+                                        }
+                                    }
+
+                                    return false;
+                                })
+                                ->thenInvalid('Only "apcu", "doctrine", "filesystem", "psr6" or "redis" cache adapter names can have no "parent" configuration key.')
+                            ->end()
+                        ->end()
                         ->arrayNode('pools')
                             ->useAttributeAsKey('name')
                             ->prototype('array')
                                 ->children()
                                     ->scalarNode('adapter')
-                                        ->info('The cache pool adapter service to use as template definition.')
+                                        ->info('The cache adapter service to use as template definition.')
                                         ->defaultValue('cache.adapter.shared')
                                     ->end()
                                     ->booleanNode('public')->defaultFalse()->end()
```

Comment:
I'm wondering if we should change the default to true to be consistent with the default for services